Library/Homebrew/rubocops/lines.rb

```diff
@@ -750,18 +790,6 @@ def audit_formula(formula_nodes)
             problem "Use ruby-macho instead of calling #{@offensive_node.source}"
           end
 
-          find_every_method_call_by_name(body_node, :system).each do |method_node|
-            # Skip Kibana: npm cache edge (see formula for more details)
```

Comment:
this was staled, we removed kibana last year: https://github.com/Homebrew/homebrew-core/commit/39a6532cc2b